<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="title">
     <string>history</string>
    </property>
    <widget class="QTextEdit" name="ui_history_TE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui_preorderAmount">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>8</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>45</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemCodeLabel_3">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>장전매수수량</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ui_sb_realtrade">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>장전매수</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="title">
     <string>전일정보</string>
    </property>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>미국 S&amp;P 500</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_us500_LE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>45</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_daw30_LE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>20</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>미국 다우 산업</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>미국 나스닥</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_nasdaq_LE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>70</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>마지막 거래일</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_uslastday_LE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>95</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>코스피 전일가</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>155</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>코스닥 전일가</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_kospi_LE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>130</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_kosdaq_LE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>155</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>마지막 거래일</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_kolastday_LE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>180</y>
       <width>190</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>691</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="title">
     <string>Balance &amp;&amp; Current stock status</string>
    </property>
    <widget class="QTableWidget" name="ui_balance_TW">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>671</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>105</number>
     </attribute>
     <row/>
     <column>
      <property name="text">
       <string>예수금(d+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="ui_stockstatus_TW">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>671</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>92</number>
     </attribute>
     <column>
      <property name="text">
       <string>종목코드</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률(%)</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="ui_realtime_CB">
     <property name="geometry">
      <rect>
       <x>521</x>
       <y>235</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>Real Time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_check_PB">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>230</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>144</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>204</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>174</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_accountlist_CB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_hoga_CB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="ui_ordertype_CB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="ui_stockcode_LE">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_stockname_LE">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ui_num_SB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="ui_price_SB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_sendorder_PB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>611</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>종목명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>전일가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>시작가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>현재가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>최고가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>최저가</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuori">
    <property name="title">
     <string>ori</string>
    </property>
   </widget>
   <addaction name="menuori"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
